tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,33 @@
 """
 
 
+test_file = """
+#include "CPPCleanBearHeaderTest.h"
+
+int main() {
+    cppcleanbearfunction();
+    return 0;
+}
+"""
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(),
+    invalid_files=(test_file,),
+    tempfile_kwargs={'suffix': '.cpp'}
+    )
+
+CPPCleanBearIncludePathTest2 = verify_local_bear(
```

Comment:
hmm bad name... Rather call it `CPPCleanBearValidIncludePathTest` (and above `CPPCleanBearNoIncludePathTest) or similar. Test names should always reflect their purpose / functionality tested.